### DISCUSSION TITLE
Query about dangerzone and poetry shell

### DISCUSSION THREAD

**Discussion by mc-gitdeb** [freedomofpress/dangerzone#586] [General]
Hi, i'm wondering something,

Does dangerzone cli have to be run within a poetry shell ideally?

I have got it to work by just changing to the dangerzone folder and running it straight from there,

But when I try to run it within poetry shell i get a big error saying file not found and other stuff

Sorry, i'm not used to building from source and virtual env, but i do have a linux sysadmin course done so I know some things

**Comment by deeplow** [comment]
Sorry for the delay.

> Does dangerzone cli have to be run within a poetry shell ideally?

you run it with `poetry run python ./dev_scripts/dangerzone-cli`
